packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -1041,6 +1041,18 @@ async function startWatcher(opts: SetupOpts) {
       )
     }
 
+    async function writeManifests(): Promise<void> {
```

Comment:
![Is This A Pigeon](https://github.com/vercel/next.js/assets/11064311/5ee1c5b6-881e-4318-8bb3-b5ffb9d6cfaa)